openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/test/sc/LoaderSlaveMultiCL.java

```diff
@@ -91,8 +100,18 @@ public void run(String jarName)
 				{
 					cntr++;
 					Dummy myDummy = (Dummy)myC.newInstance();
-					if(cntr % 1000 == 0)
+					if(cntr % 1000 == 0) {
 						logMessage("Loaded " + cntr + " classes...");
+						
+						// This flag has been added to terminate the load test within 10 minutes 
+						// keeping in mind that the test launches 5 processes to simultaneously run this load and 
+						// the overall time it takes is especially slow on certain platforms (e.g. Windows). 		
+						long durationInMinutes = (System.currentTimeMillis() - startTime) / (1000 * 60);  
+						if (durationInMinutes > 10 && isWindows) {
```

Comment:
Why does this need to be restricted to Windows? Other platforms are running slowly as well, so might as well do this for all platforms.

Reply:
From the [findings](https://github.com/eclipse/openj9-systemtest/pull/99#issuecomment-535156261) above  - it appears it's only running slow on Windows. The rest are reasonably fast.

Reply:
> The rest are reasonably fast.

On the particular machines you looked at. It may not be true for everybody running the tests on whatever machines. If we can limit to 10 min on Windows, it might as well be limited for all.

Reply:
Makes sense. Check for Windows has been removed. The load will be 10m for all platforms.

Reply:
I've also realized this isn't a Windows problem or test problem, it results from a change to the VM or JIT.

Reply:
OK. The change is ready.  Please merge it when you feel appropriate.

Reply:
Given we've discovered this was a code problem, I think we should take a different approach. The test should finish in about 10 min normally (see https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.system_x86-64_windows_Nightly/139/tapTestReport/), but the timeout is 2 hours. I think we should scale back the timeout to 20 min and see how that goes.

Reply:
Even after we reduce the load, the test took [more than 1 hour](https://hyc-runtimes-jenkins.swg-devops.com/view/Test_grinder/job/Grinder_Advanced/1717/) (on Windows) to complete. Note that part of this time is spent in generating classes and creating jar files that are later used in the classloading phase of the test. 

If we set timeout to 20m - we need to reduce the amount of work that happens in the class file creation and jar generation phase by about 1/5th of what it does now. 

This means reducing the workload on all platforms other than Windows too - where the current workload works just fine (e.g. they all finished in less than 5 minutes in the previous nightly run).